package/gluon-client-bridge/luasrc/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -24,7 +23,7 @@ local function configure_ap(radio, index, config, radio_name)
 	local ap = config.ap
 	local disabled = is_disabled(ap, name)
 
-	if not util.supports_channel(radio, config.channel()) then
+	if not wireless.supports_channel(radio, config.channel()) then
```

Comment:
Another commit that needs to be squashed. Always squash fixes, so the resulting commits make sense by themselves for review.

Reply:
I left the commits splitted, as parts were developed by blocktrron and I wanted to keep this state - but I squashed it now, for easier reviewing

Reply:
In that case "co-authored-by" should be added to the commits this affects.